$NetBSD: patch-ac,v 1.3 2009/08/09 08:20:27 wiz Exp $

LYMain.c: In function 'SetLocale':
LYMain.c:941: error: 'LC_ALL' undeclared (first use in this function)
LYMain.c:941: error: (Each undeclared identifier is reported only once
LYMain.c:941: error: for each function it appears in.)

--- src/LYMain.c.orig	2009-07-05 23:52:33.000000000 +0000
+++ src/LYMain.c
@@ -57,7 +57,7 @@
 #include <io.h>
 #endif
 
-#if defined(LOCALE) && !defined(HAVE_LIBINTL_H)
+#if defined(LOCALE)
 #undef gettext			/* Solaris locale.h prototypes gettext() */
 #include <locale.h>
 #ifndef HAVE_GETTEXT
